static/js/directives/audiovideo.js

```diff
@@ -28,7 +28,7 @@ define(['jquery', 'underscore', 'text!partials/audiovideo.html', 'text!partials/
 
 		var controller = ['$scope', '$element', '$attrs', function($scope, $element, $attrs) {
 
-			var streams = {};
+			var streams = this.streams = {};
```

Comment:
[Here](https://github.com/theurere/spreed-webrtc/blob/update-to-angular-1.3/static/js/directives/audiovideo.js#L423) you pass the controller with stream property to `videoLayout.update`.